samples/Kconfig.mcumgr_bt_ota_dfu

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr and OTA DFU over Bluetooth"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	select MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	select MCUMGR_CMD_IMG_MGMT
+	select MCUMGR_CMD_OS_MGMT
+	select MCUMGR_GRP_ZEPHYR_BASIC
+	select MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
+	select MCUMGR_SMP_REASSEMBLY_BT
+	select NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	select OS_MGMT_MCUMGR_PARAMS
```

Comment:
Also select or imply for the options related to speedup?

Reply:
I will define it as a default to highlight that this only needed for the Reassembly feature (similar dependency for MCUmgr buffer size)